### ISSUE TITLE
I can configure "exposed" values on a profile containing raw yaml

### ISSUE THREAD

**Issue by Callisto13** [weaveworks/profiles#36]
something along the lines of:

```
$ pctl show weaveworks/nginx-ingress
Version: 1.10.1
Catalog: weaveworks (https://github.com/weaveworks/profiles)
Prerequisites: Kubernetes 1.18+
Description: NGINX Ingress Controller
Maintainer: WeaveWorks <gitops@weave.works>
Configurable values: containerPort, somethingElse...
```

_not exact api, just illustrative_
```
apiVersion: weave.works/v1alpha1
kind: ProfileSubscription
metadata:
  name: nginx-profile-test
  namespace: default
spec:
  profileURL: https://github.com/weaveworks/nginx-profile
  branch: main
  values:
     containerPort: 8080
     somethingElse: foo
```

**Comment by aclevername** [comment]
closing due to inactivity. Feel free to re-open or create a new issue with more up to date info :smile:
